aiopg/pool.py

```diff
@@ -187,6 +190,10 @@ def _fill_free_pool(self, override_min):
             conn = self._free[-1]
             if conn.closed:
                 self._free.pop()
+            elif self._recycle > -1 \
+                    and time.time() - conn.last_usage > self._recycle:
```

Comment:
Should be `self._loop.time()`: asyncio uses `time.monotonic()` internally

Reply:
@asvetlov Done